src/main/java/io/reactivex/Completable.java

```diff
@@ -943,10 +943,9 @@ public final Completable delay(final long delay, final TimeUnit unit, final Sche
      * @throws NullPointerException if onComplete is null
      */
     @SchedulerSupport(SchedulerSupport.NONE)
-    public final Completable doOnComplete(Action onComplete) {
-        return doOnLifecycle(Functions.emptyConsumer(), Functions.emptyConsumer(),
-                onComplete, Functions.EMPTY_ACTION,
-                Functions.EMPTY_ACTION, Functions.EMPTY_ACTION);
+    public final Completable doOnComplete(final Action onComplete) {
```

Comment:
Looks like that comment was misaligned. You changed `SingleDoOnEvent` to have 2 arguments and then had 2 allocations to work with `BiConsumer`.